trl/trainer/ppo_trainer.py

```diff
@@ -793,3 +839,23 @@ def log_stats(
 
                 dist.barrier()
                 dist.all_reduce(rewards, op=torch.distributed.ReduceOp.SUM)
+
+    def create_model_card(self, path: str, model_name: Optional[str] = "TRL Model") -> None:
+        """Creates and saves a model card for a TRL model.
+
+        Args:
+            path (`str`): The path to save the model card to.
+            model_name (`str`, *optional*): The name of the model, defaults to `TRL Model`.
+        """
+        if not os.path.exists(path):
+            os.makedirs(path)
+
+        user = whoami()["name"]
```

Comment:
This line is the reason why the unit test in the staging endpoint fails - by default `whoami()` points to hf.co, when we need it to point to https://hub-ci.huggingface.co

One can do this by using `HfApi(endpoint="https://hub-ci.huggingface.co")`, but then we need to set an env var for `endpoint` which feels clunky. 

For now, I propose we skip the unit test until we find a better solution to test `push_to_hub` features

OK with you @younesbelkada @lvwerra ?

Reply:
One way to solve this is by patching the function in the test? we could have a dummy function for whoami that returns whatever we want during the testing? 

happy to merge it as is and figure it out later :)

Reply:
same as @lvwerra , definitely OK for me to skip this test for now and look at it later!